core/frame/meta_frame.go

```diff
@@ -1,43 +1,62 @@
 package frame
 
 import (
-	"strconv"
 	"time"
 
 	"github.com/yomorun/y3"
 )
 
-// MetaFrame is a Y3 encoded bytes, SeqID is a fixed value of TYPE_ID_TRANSACTION.
-// used for describes metadata for a DataFrame.
+const (
+	// LoadBalanceRandomPick means Zipper will choose SFN instance by a load balance strategy.
+	LoadBalanceRandomPick byte = 0x00
+	// LoadBalanceBindInstance means data will be sent to the specific SFN according to instanceID.
+	LoadBalanceBindInstance byte = 0x01
```

Comment:
naming can reference this: https://www.cloudflare.com/learning/performance/types-of-load-balancing-algorithms/

Reply:
these are specific means, not the target